protocol/dubbo3/dubbo3_protocol.go

```diff
@@ -99,18 +99,18 @@ func (dp *DubboProtocol) Export(invoker protocol.Invoker) protocol.Exporter {
 		valueOf := reflect.ValueOf(service)
 		typeOf := valueOf.Type()
 		numField := valueOf.NumMethod()
-		tripleService := &Dubbo3HessianService{proxyImpl: invoker}
+		tripleService := &UnaryService{proxyImpl: invoker}
 		for i := 0; i < numField; i++ {
 			ft := typeOf.Method(i)
 			if ft.Name == "Reference" {
 				continue
 			}
-			// num out is checked in common/rpc_service.go
-			if ft.Type.NumIn() != 3 {
```

Comment:
why delete the check cond?

Reply:
Thisk Block limit the number of method input params, if not 1 param, it would panic.
Now, we support any number of param.

Reply:
awesome.